tree/ntuple/v7/doc/specifications.md

```diff
@@ -1,7 +1,36 @@
-# RNTuple Reference Specifications (WIP)
+# RNTuple Reference Specifications 0.2.0.0
 
 **Note:** This is work in progress. The RNTuple specification is not yet finalized.
 
+## Versioning Notes
+
+The RNTuple binary format vesion is inspired by semantic versioning.
+It uses the following scheme: EPOCH.MAJOR.MINOR.PATCH
+
+_Epoch_: an increment of the epoch indicates backwards-incompatible changes.
```

Comment:
When do we envision a backwards-incompatible change? If I'm reading this correctly it means at some data will require re-processing at that point?

(similar to https://arrow.apache.org/docs/format/Versioning.html#long-term-stability, except we have an additional level "epoch")

Reply:
After version 1.0, no backwards-incompatible change is foreseen. But we wanted to prepare the versioning for it anyway, just in case.